monitoring/api/AlertSample/README.md

```diff
@@ -0,0 +1,61 @@
+# Google Stackdriver Monitoring Sample
+
+This C# sample demonstrates how to manage Stackdriver Alerts using
+[Google Stackdriver Monitoring API](https://cloud.google.com/monitoring/alerts/using-alerting-api) from C#.
+
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+    https://www.visualstudio.com/), or the command line.  Visual Studio 2015 users
+can use [this older sample](
+    https://github.com/GoogleCloudPlatform/dotnet-docs-samples/tree/vs2015/monitoring/api).
+
+## Build and Run
+
+1.  **Follow the instructions in the [root README](../../README.md)**.

```

Comment:
Update markdown href path 
    [root README](../../README.md)
to be
    [root README](../../../README.md)